ui/src/main/java/be/appwise/ui/EditText.kt

```diff
@@ -0,0 +1,730 @@
+                            Thumb(
+                                interactionSource = interactionSource
+                            )
+                        }
+
+                    }
+                },
+                track = remember(colors, enabled) {
+                    { sliderPositions ->
+                        SliderDefaults.Track(
+                            colors = colors,
+                            enabled = enabled,
+                            sliderPositions = sliderPositions
+                        )
+                    }
+                })
+        }
+
+        Message(message = message, icon = icon)
+
+    }
+}
+
+/**
+ *
+ * Radio buttons allow users to select one option from a set.
+ *
+ * @param modifier the [Modifier] to be applied to this radio button
+ * @param onOptionSelected callback in which value should be updated, the other values automatically gets updated to false
+ * @param values expects a [List] of strings where the string is the label value.
+ * @param enabled controls the enabled state of this radio button. When `false`, this component will
+ * not respond to user input, and it will appear visually disabled and disabled to accessibility
+ * services.
+ * @param colors [RadioButtonColors] that will be used to resolve the color used for this radio
+ * button in different states. See [RadioButtonDefaults.colors].
+ * [Interaction]s and customize the appearance / behavior of this radio button in different states.
+ * @param icon the optional icon to be displayed in front of the message or error message. Default value is Icons.Outlined.ErrorOutLine.
+ * @param message the optional error/info message. The message is shown underneath the text field,
+ * when isError is true this message will be an error message and shown in ErrorColor from the appTheme.
+ * @param isRequired when `true`, the input will show a * behind the label in the Red color.
+ * @param label the optional label to be displayed above the text field container.
+ *
+ */
+
+@Composable
+fun EditTextRadioButton(
+    modifier: Modifier = Modifier,
+    onOptionSelected: (String) -> Unit = {},
+    values: List<String>,
+    label: String,
+    message: String = "",
+    icon: ImageVector = Icons.Outlined.ErrorOutline,
+    isRequired: Boolean = false,
+    colors: RadioButtonColors = RadioButtonDefaults.colors(selectedColor = MaterialTheme.colorScheme.primary),
+    enabled: Boolean = true
+) {
+
+    val map = remember {
+        mutableStateMapOf(*values.map { it to false }.toTypedArray())
+    }
+
+    Column {
+        Label(label = label, isRequired = isRequired)
+
+        Column(
+            modifier = Modifier.selectableGroup()
+        ) {
+            map.forEach { radioButton ->
+                Row(
+                    modifier = modifier
+                        .fillMaxWidth()
+                        .padding(bottom = 5.dp)
+                        .background(color = Color.Gray, shape = RoundedCornerShape(10.dp)),
```

Comment:
I would move this backgroundColor + shape to the parameters list of the function so we can modify it to match any design

Reply:
@senne-vanreusel I think you forgot to move this BackgroundColor and Shape to the params of the Composable